sdk/identity/azure-identity/azure/identity/_credentials/azd_cli.py

```diff
@@ -101,6 +107,7 @@ def get_token(self, *scopes: str, **kwargs: Any) -> AccessToken:
             For more information about scopes, see
             https://learn.microsoft.com/azure/active-directory/develop/scopes-oidc.
         :keyword str tenant_id: optional tenant to include in the token request.
+        :keyword str claims: not used by this credential; any value provided will be ignored.
```

Comment:
Hmm, I feel that it is more clear and consistent if the the keyword argument docstrings appear in the same order as they appear in the method signature (even if one that is not used appears ahead of one that is used).

Reply:
That's fine.

We don't have a strict ordering requirement. :)

Reply:
On second thought, I'll make these consistent with the signature since arguments aren't actually ignored very frequently. My initial worry was with how customers would react to seeing `claims` ahead of `tenant_id` in docstrings if it was being tossed 90% of the time -- if it's an exception rather than the rule, it doesn't seem as weird to me.